exotica_py/package.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>exotica_py</name>
```

Comment:
These naming conventions relate to package names, not Python module names (PEP-8 supports underscores if they both are whole words and readability is improved). ``py`` is not a whole word. After thinking things, through, perhaps the following is the best:

- ROS package ``exotica_python`` and Python module name ``exotica`` (drop the py or python from the module name - as a suffix it looks very ugly, as a prefix would look better)

Perhaps let's have a chat F2F over lunch?